### PR TITLE
Update conda-forge to more closley mirror gunicorn's

### PR DESCRIPTION
**Pull request by corydolphin**
Apologies for the noise. This is an accidental change, I had intended to push this up to my fork. I'm working on trying to enable Python 3.11 builds (unsuccesfully).

### CODE REVIEW

**Comment by corydolphin** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,14 +16,17 @@ build:
...
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
     - icu
     - jansson
     - libpython-static  # [py>=38]
     - libxml2  # [not win]
-    - openssl
+    - openssl  # [not linux]
+    - openssl =1.1.1  # [linux]
```
Comment (jakirkham) [inline recipe/meta.yaml:29]: This shouldn't be needed. Would drop it

**Comment by jakirkham** [comment]
Please make sure to re-render after adding cross-compilation pieces

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
 MACOSX_SDK_VERSION:            # [osx and x86_64]
-  - 10.15                      # [osx and x86_64]
+  - 11.0                       # [osx and x86_64]
```
Comment (jakirkham) [inline recipe/conda_build_config.yaml:2]: Please read [this]( https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks )

**Pull request closed without merging**